docs/api-guide/settings.md

```diff
@@ -103,6 +103,13 @@ A view inspector class that will be used for schema generation.
 
 Default: `'rest_framework.schemas.openapi.AutoSchema'`
 
+#### DEFAULT_MISSING_DATA
```

Comment:
Maybe let's call this something like `EMPTY_REQUEST_DATA`?

The `DEFAULT_XYZ` cases make sense because they're things that *can be overridden*, and are populating the default value. In this case it's not a default case that can be overridden, but rather the base case that it always used if there's an empty request.

Reply:
Happy to take feedback on alternate naming choices, too.

Reply:
I chose "missing" as opposed to "empty" because conventional usage of the word "empty" would suggest that a request body of `""`, `[]` or `{}` is also empty (`len(request.data) == 0` after deserialization). I feared that if we call it "EMPTY_DATA", one may think that any "empty-like" payload would be coerced into this value. -- However, I don't feel strongly, as long as the documentation is clear.

Regarding whether it is a default: If by "override", you mean overriding by view attribute or similar, fair enough; one could also argue that the client can override it (by sending data), which would warrant the name `DEFAULT_DATA` or `DEFAULT_REQUEST_DATA`.

(Also, it's conceivable that there would a view attribute to override this setting at some point, although right now the request would not have the corresponding context.)

Having said all this, I think my favorite would be `DEFAULT_REQUEST_DATA`, as I think it is accurate and future-proof. But in the end, I don't have any stakes in the naming, and will be ok with whatever you prefer.